### PR TITLE
chore: gitworkflows

### PR DESCRIPTION
**Pull request by nkoppisetty**
Git workflows for changelog, pypi and coverage

### CODE REVIEW

**Review by vigith — APPROVED** [review]
LGTM

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 1%
```
Comment (ab93) [inline .codecov.yml:6]: let's make this 10%
```suggestion
        threshold: 10%
```

File: .codecov.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    patch:
+      default:
+        target: auto
+        threshold: 1%
```
Comment (ab93) [inline .codecov.yml:10]:
```suggestion
        threshold: 20%
```

**Review by ab93 — COMMENTED** [review]
minor comments

**Review by ab93 — APPROVED** [review]

**Pull request merged**